page-components/featured-content-module.md

```diff
@@ -110,11 +94,11 @@ Differences in style at tablet size
 
 Differences in style at mobile size
 
-* Module appears 30px underneath the header/intro paragraph of the page
-* Headline: H3
-* Body copy: 16px
+* Height is variable depending on how much content is in the FCM
```

Comment:
This is not different from the wider layout, and would probably be assumed by most, anyway.

Reply:
Agreed. I reworked this a little in eeb539d to put all the info about text and visual stacking in one bullet.